run/GCHP/runScriptSamples/operational_examples/harvard_cannon/gchp.run

```diff
@@ -28,11 +28,9 @@ export OMPI_MCL_btl=openib
 # Run GCHP and evenly distribute tasks across nodes
 NX=$( grep NX GCHP.rc | awk '{print $2}' )
 NY=$( grep NY GCHP.rc | awk '{print $2}' )
-coreCount=$(( ${NX} * ${NY} ))
-planeCount=$(( ${coreCount} / ${SLURM_NNODES} ))
-if [[ $(( ${coreCount} % ${SLURM_NNODES} )) > 0 ]]; then
-	${planeCount}=$(( ${planeCount} + 1 ))
-fi
+let coreCount=$(( ${NX} * ${NY} ))
```

Comment:
Is adding let necessary? It looks like otherwise the line is the same, and I don't think this was the line causing the issue. Same comment for the next line (planeCount).